src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -447,7 +448,7 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
                 ResolutionFailure::NotResolved {
                     module_id,
                     partial_res: None,
-                    unresolved: item_str.into(),
+                    unresolved: path_str.into(),
```

Comment:
Why did this change? Before it would only report `x` as unresolved, now it will report all of `x::y`.

Reply:
You're right. I was trying to get this working with `split_once`, and left this from that experiment accidentally. Thanks!